xonsh/built_ins.py

```diff
@@ -905,6 +905,8 @@ def run_subproc(cmds, captured=False):
     if _should_set_title(captured=captured):
         # set title here to get currently executing command
         pause_call_resume(proc, builtins.__xonsh__.shell.settitle)
+    else:
+        pause_call_resume(proc, lambda : 0)
```

Comment:
A couple of small things:

1. Can you put some comments above this line that discuss why this case is here. I think it might be confusing otherwise.
2. can you replace the `lambda: 0` with `int`, please.  `int()` [with no arguments] returns zero and is already defined.

Thanks!

Reply:
@scopatz done